format/index.md

```diff
@@ -472,14 +488,14 @@ multiple times.
 
 As discussed above, the document **MAY** be extended to include
 meta-information as `"meta"` members in several locations: at the top-level,
-within resource objects, within link objects, and within linkage objects.
+within resource objects, within relationship objects, and within linkage objects.
```

Comment:
As I mentioned off-handedly in #618, I think there's an argument for renaming "linkage objects" to "resource identifier objects". This is a bit longer, but it emphasises very nicely the spec’s concept that type/id is used to uniquely identify a resource, and it allows us to (more sensibly) refer to objects of this form where we’re using them in contexts other than the `"linkage"` key, such as for bulk deletion in the bulk extension.

If you guys agree, I can make a separate PR for that, since I don't want this one to get too unwieldy.